metadata.json

```diff
@@ -14,9 +14,9 @@
   ],
   "operatingsystem_support": [
     {
-      "operatingsystem": "Netscaler 10.6",
+      "operatingsystem": "Netscaler 10.5",
```

Comment:
This is just a nitpick, but shouldn't this just say `Netscaler` and allow the `operatingsystemrelease` string below handle the rest?